### PR TITLE
fix: !!! info notation correction

### PR DESCRIPTION
**Pull request by KhalilSelyan**
## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The Reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: docs/installation/autoware/source-installation.md [hunk 1]
```diff
@@ -65,9 +65,10 @@ sudo apt-get -y install git
 
 ## How to set up a workspace
 
-!!! info [Using Autoware Build GUI](#using-autoware-build-gui)
+!!! info
```
Comment (xmfcx) [inline docs/installation/autoware/source-installation.md:68]:
```suggestion
!!! info "[Using Autoware Build GUI](#using-autoware-build-gui)"
```

Can we test like this instead?

File: docs/installation/autoware/source-installation.md [hunk 2]
```diff
@@ -65,9 +65,10 @@ sudo apt-get -y install git
...
 
-    If you prefer a graphical user interface (GUI) over the command line for launching and managing your simulations, refer to the `Using Autoware Build GUI` section at the end of this document for a step-by-step guide.
+      [Using Autoware Build GUI](#using-autoware-build-gui)
+      If you prefer a graphical user interface (GUI) over the command line for launching and managing your simulations, refer to the `Using Autoware Build GUI` section at the end of this document for a step-by-step guide.
```
Comment (xmfcx) [inline docs/installation/autoware/source-installation.md:71]:
```suggestion
      If you prefer a graphical user interface (GUI) over the command line for launching and managing your simulations, refer to the `Using Autoware Build GUI` section at the end of this document for a step-by-step guide.
```
continuation of suggestion above

File: docs/installation/autoware/source-installation.md [hunk 3]
```diff
@@ -65,9 +65,9 @@ sudo apt-get -y install git
 
 ## How to set up a workspace
 
-!!! info [Using Autoware Build GUI](#using-autoware-build-gui)
+!!! info "[Using Autoware Build GUI](#using-autoware-build-gui)"
 
-    If you prefer a graphical user interface (GUI) over the command line for launching and managing your simulations, refer to the `Using Autoware Build GUI` section at the end of this document for a step-by-step guide.
+      If you prefer a graphical user interface (GUI) over the command line for launching and managing your simulations, refer to the `Using Autoware Build GUI` section at the end of this document for a step-by-step guide.
```
Comment (xmfcx) [inline docs/installation/autoware/source-installation.md:70]:
```suggestion
     If you prefer a graphical user interface (GUI) over the command line for launching and managing your simulations, refer to the `Using Autoware Build GUI` section at the end of this document for a step-by-step guide.
```

Where do these whitespaces come from? :confused:

Comment (KhalilSelyan) [reply docs/installation/autoware/source-installation.md:70]: I'm not entirely sure, but i'll just take the commit in and hope it stays that way

**Review by xmfcx — APPROVED** [review]

**Pull request merged**